src/backend/storage/file/fd.c

```diff
@@ -2663,14 +2687,27 @@ GetSessionTempTableSpace(void)
 	if (gp_session_id >= 0)
 		return tempTableSpaces[gp_session_id % numTempTableSpaces];
 
-	/* If this session is not MPP, uses the implementation from upstream */
```

Comment:
I think, you should not move this comment

Reply:
Done